packages/documentation/pages/foundations/colors/list-of-colors.vue

```diff
@@ -81,7 +81,7 @@ import ColorPalette from '~/components/ColorPalette.vue'
 import ColorUsage from '~/components/ColorUsage.vue'
 
 export default {
-	name: 'Color',
+	name: 'ListOfColors',
```

Comment:
Should follow the naming strategy the other doc pages follow (something like `DocumentationPageFoundationsColor...`)